### ISSUE TITLE
After hashing files some files "dont exist"

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dizzydezz/jmm#434]
```
Hey I have a crappy problem,i dont know if mySQL is doing this or what but For 
example I add 
anime,anime2,anime3
and then anime 4 then in jmm desktop i go to collection click on the anime that 
is from anime 4,try to play the file but it says file does not exist,all files 
added from anime 4 "does not exist",i then created a new database and added 
anime then anime 4,then anime 2 and anime 3 and now anime 2,3 does this.When I 
hover my mouse on the episode tab the location of the hashed file is wrong for 
example the file is at C:\\anime2\.hack but it says the file is not found and 
the location that the program has the file is c:\\anime\2\.hack and this is 
wrong
```

Original issue reported on code.google.com by `DDVers...@gmail.com` on 12 May 2013 at 9:24

**Comment by GoogleCodeExporter** [comment]
```
Ok, I will set up a test with file names as you describe.

It looks like it is re-parsing some folder names. Probably due to the full stop 
in the folder name
```

Original comment by `werndly...@gmail.com` on 12 May 2013 at 11:30
- Changed state: **Accepted**
- Added labels: **Priority-High**, **Product-JMM**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
Hello 
I originally had anime 1,2,3,4 but now i have anime and anime 2 but anime 2 
still does the same.Anyway the full stop is only in one anime .hack but not any 
other folder name,i've attatched screen shots of jmm desktop and i've printed 
my entire anime collection that shows the names and everything hope this helps 
:).
```

Original comment by `DDVers...@gmail.com` on 13 May 2013 at 1:33

Attachments:
- [this is anime 2 but it looks in the wrong directory as you can see. and it does not exist so the hasher or something does this i dont know if this is mySQL.png](https://storage.googleapis.com/google-code-attachments/jmm/issue-434/comment-2/this is anime 2 but it looks in the wrong directory as you can see. and it does not exist so the hasher or something does this i dont know if this is mySQL.png)
- [this is the folder anime and every file works.png](https://storage.googleapis.com/google-code-attachments/jmm/issue-434/comment-2/this is the folder anime and every file works.png)
- [this is all my directories.txt](https://storage.googleapis.com/google-code-attachments/jmm/issue-434/comment-2/this is all my directories.txt)

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 3.1.32
```

Original comment by `werndly...@gmail.com` on 28 May 2013 at 12:14
- Changed state: **Fixed**
